diffprivlib/mechanisms/exponential.py

```diff
@@ -193,7 +193,7 @@ def _get_utility(self, value1, value2):
             return 0
 
         if value1 > value2:
-            return self._get_utility(value2, value1)
+            return self._get_utility(value1=value2, value2=value1)
 
         return self._utility_values[value1 + "::" + value2]
```

Comment:
Suggestion to replace string keys with tuples

Reply:
Thanks for the suggestion, it has been fixed in https://github.com/IBM/differential-privacy-library/commit/6b3183bd3dd807c4e2c3416358c0b95c875692fd